app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/driveBar.tsx

```diff
@@ -69,27 +69,26 @@ export default function DriveBar(props: DriveBarProps) {
                 </button>
                 <div className="flex items-center text-black-a50 bg-frame rounded-full gap-3 h-8 px-3">
                     {backend.type !== backendModule.BackendType.local && (
-                        <>
-                            <Button
-                                active
-                                image={AddFolderIcon}
-                                disabledOpacityClassName="opacity-20"
-                                onClick={() => {
-                                    unsetModal()
-                                    doCreateDirectory()
-                                }}
-                            />
-                            <Button
-                                active
-                                disabled
-                                image={AddConnectorIcon}
-                                error="Not implemented yet."
-                                disabledOpacityClassName="opacity-20"
-                                onClick={() => {
-                                    // No backend support yet.
-                                }}
-                            />
-                        </>
+                        <Button
+                            active
+                            image={AddFolderIcon}
+                            disabledOpacityClassName="opacity-20"
+                            onClick={() => {
+                                unsetModal()
+                                doCreateDirectory()
+                            }}
+                        />
+                    )}
+                    {backend.type !== backendModule.BackendType.local && (
+                        <Button
+                            disabled
+                            image={AddConnectorIcon}
+                            error="Not implemented yet."
+                            disabledOpacityClassName="opacity-20"
+                            onClick={() => {
+                                // No backend support yet.
```

Comment:
i *think* it's fine to not open an issue for this? if it's something that should be implemented on the backend then i think the backend parts will be planned separately? because there are quite a number of features that do currently lack backend support, so i'm just not sure it's a good idea to file everything as issues